vision/transforms/to_tensor.go

```diff
@@ -30,27 +30,29 @@ func (t ToTensorTransformer) Run(obj interface{}) torch.Tensor {
 
 // ToTensor transform c.f. https://github.com/pytorch/vision/blob/ba1b22125723f3719a3c38a2fe7cd6fb77657c57/torchvision/transforms/functional.py#L45
 func imageToTensor(img image.Image) torch.Tensor {
-	width, height := img.Bounds().Max.X, img.Bounds().Max.Y
+	w, h := img.Bounds().Max.X, img.Bounds().Max.Y
 	// put pixel values with HWC format
-	array := make([][][3]float32, height)
-
-	for x := 0; x < height; x++ {
-		row := make([][3]float32, width)
-		for y := 0; y < width; y++ {
-			// ResNet need the 3 channels image, here we should convert to RGB format.
-			// The division by 255.0 is applied to convert RGB pixel values from [0, 255] to [0.0, 1.0] range
-			c := img.(*image.NRGBA).NRGBAAt(x, y)
-			row[y] = [3]float32{float32(c.R) / 255.0, float32(c.G) / 255.0, float32(c.B) / 255.0}
+	array := make([]float32, h*w*3)
+	denom := float32(0xffff)
```

Comment:
Should be `float32(0xff)`, `float32(0xffff)` is 65535.

Reply:
It is 0xffff. Not 0xff